synapse/models/biz.py

```diff
@@ -0,0 +1,204 @@
+import synapse.lib.module as s_module
+
+'''
+Model elements related to sales / bizdev / procurement
+'''
+
+class BizModule(s_module.CoreModule):
+    def getModelDefs(self):
+        modl = {
+            'types': (
+                ('biz:rfp', ('guid', {}), {
+                    'doc': 'An RFP (Request for Proposal) soliciting proposals.',
+                }),
+                ('biz:deal', ('guid', {}), {
+                    'doc': 'A sales or procurement effort in pursuit of a purchase.',
+                }),
+                ('biz:bundle', ('guid', {}), {
+                    'doc': 'Instances of a specific product offered for a price.',
+                }),
+                ('biz:product', ('guid', {}), {
+                    'doc': 'A product which is available for purchase.',
+                }),
+                ('biz:dealstatus', ('taxonomy', {}), {
+                    'doc': 'A deal/rfp status taxonomy.',
+                    'interfaces': ('taxonomy',),
+                }),
+                ('biz:dealtype', ('taxonomy', {}), {
+                    'doc': 'A deal type taxonomy.',
+                    'interfaces': ('taxonomy',),
+                }),
+                ('biz:prodtype', ('taxonomy', {}), {
+                    'doc': 'A product type taxonomy.',
+                    'interfaces': ('taxonomy',),
+                }),
+            ),
+            'forms': (
+                ('biz:dealtype', {}, ()),
+                ('biz:prodtype', {}, ()),
+                ('biz:dealstatus', {}, ()),
+                ('biz:rfp', {}, (
```

Comment:
Do we want to add a :proposal prop to capture  biz:proposal nodes that were submitted in response to the RFP?

Reply:
This would be an `:rfp` secondary prop on the proposal node so remains TBD